code/front/js/productFront.js

```diff
@@ -0,0 +1,148 @@
+/*
+** single product page
+*/
+
+//get _id of product from query parameters
+const queryString = window.location.search;
+console.log(queryString);
+let splitForId = queryString.slice(5);
+console.log(splitForId);
+
+const newUri = 'http://localhost:3000/api/products/' + splitForId;
+let product;
+
+/*
+** fetch
+*/
+
+//fetch the new URI from get request
+fetch(newUri) 
```

Comment:
`fetch(PRODUCT_URI)`

Reply:
This was resolved but didn't successfully appear in the git push to dev branch, Perhaps I should hit command+s first on the files